shared/js/ui/templates/shared/tracker-networks-text.es6.js

```diff
@@ -17,7 +17,7 @@ module.exports = function (site, isMajorNetworksCount) {
 
 function trackersBlockedOrFound (site, trackersCount) {
     let msg = ''
-    if (site && (site.isWhitelisted || trackersCount === 0)) {
+    if (site && site.isAllowlisted || trackersCount === 0) {
```

Comment:
So this means that when `site` is falsy we also say "Found", while before we said "Blocked", right? In which cases is `site` falsy? I guess one example might be when the extension reloads while a tab is open. Will we not be blocking requests for the tab in that case?

Reply:
We should be at least once the tds list is loaded. I think this has the potential to be the case after install, I'm not super familiar with when we trigger `site` calculation, I think it's on navigation from memory.

I've made the brackets match how they were anyway.